auction-server/src/auction/service/verification.rs

```diff
@@ -965,18 +961,22 @@ impl Service<Svm> {
         let ata =
             get_associated_token_address(&swap_accounts.user_wallet, &spl_token::native_mint::id());
         if *account_to_close != ata {
-            return Err(RestError::BadParameters(format!(
-                "Invalid account to close in close account instruction. Expected: {:?} found: {:?}",
-                ata, account_to_close
-            )));
+            return Err(RestError::InvalidInstruction(
+                None,
+                InstructionError::InvalidCloseAccountAccountToClose {
```

Comment:
this naming seems inconsistent with 
`InvalidAmountTransferInstruction` where the name of the instruction is at the end